CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## 0.5.0 - 25th August 2020
```

Comment:
Might as well add `# Changelog` at the top

Reply:
That would mean that we won't just push to the front but within the changelog. And it's also a little redundant.
That said, I don't have a strong opinion - if you prefer it with a header I'll add one.

Reply:
What's redundant about it? Every markdown file should have a single-hash `# Header` at the top.

> That would mean that we won't just push to the front but within the changelog.

Why is this a problem?

Reply:
Not a big issue, actually. I'll add the header.